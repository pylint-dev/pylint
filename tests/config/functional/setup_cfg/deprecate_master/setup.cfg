# Test the deprecation of MASTER
[MASTER]
persistent=no
