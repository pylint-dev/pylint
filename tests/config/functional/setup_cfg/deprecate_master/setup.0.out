WARNING: The use of 'MASTER' or 'master' as configuration section for pylint has been deprecated, as it's bad practice to not start sections titles with the tool name. Please use 'pylint.main' instead.
